src/dotnet/commands/dotnet-migrate/MigrateCommandParser.cs

```diff
@@ -4,7 +4,7 @@
 using System.Linq;
 using Microsoft.DotNet.Cli.CommandLine;
 using Microsoft.DotNet.Tools.Migrate;
-using LocalizableStrings = Microsoft.DotNet.Tools.Migrate.LocalizableStrings;
+using static Microsoft.DotNet.Tools.Migrate.LocalizableStrings;
```

Comment:
Please, don't do this. We are not doing it anywhere else. Let's keep the code consistent.